components/konflux-rbac/policies/validate-rolebindings/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: disallow-tenant-groups
+spec:
+  description: |
+    Asserts that rolebindings to system:anonymous are denied in tenant namespaces
```

Comment:
this is not matching with what the test checks

Reply:
fixed in cf421e6a